test/unit/test-mutator.js

```diff
@@ -0,0 +1,1670 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AmpDocSingle} from '../../src/service/ampdoc-impl';
+import {LayoutPriority} from '../../src/layout';
+import {MutatorImpl} from '../../src/service/mutator-impl';
+import {Resource, ResourceState} from '../../src/service/resource';
+import {ResourcesImpl} from '../../src/service/resources-impl';
+import {Services} from '../../src/services';
+import {Signals} from '../../src/utils/signals';
+import {VisibilityState} from '../../src/visibility-state';
+import {installInputService} from '../../src/input';
+import {installPlatformService} from '../../src/service/platform-impl';
+import {layoutRectLtwh} from '../../src/layout-rect';
+
+/** @type {?Event|undefined} */
+const NO_EVENT = undefined;
+
+/*eslint "google-camelcase/google-camelcase": 0*/
```

Comment:
This will disable the rule for the entire file. For which line is this required? You could use an inline `eslint-disable-line` instead.

Reply:
This was for certain variable names like `unused_requestedWidth`

Is there a way we can allow it wholesale? That would certainly be better.

Updated PR to use per line override.

Reply:
Aha, in that case, you could disable the rule for just that section, similar to the second example in these docs: https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments

Reply:
Updated, thanks!